docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
   report-sink:
     image: quay.io/us-cdcgov/phdo/pstatus-report-sink:latest
     ports:
-      - "127.0.0.1:8080:8080" # Rest API listener port
+      - "127.0.0.1:${SINK_PORT:-8080}:8080" # Rest API listener port
```

Comment:
Would you mind updating the main repo readme to indicate the `SINK_PORT` env var needs to be set?  Maybe with a suggestion.  Or, is there a way to default it to 8080?

Reply:
yup this defaults it to 8080, I will update the readme as well though